scheduler/server/stateful_scheduler.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/twitter/scoot/runner"
 	"github.com/twitter/scoot/saga"
 	"github.com/twitter/scoot/scheduler/domain"
```

Comment:
should this also have an alias so the domain isn't confused as the worker domain?

Reply:
Makes sense, but here, since we're aliasing the worker domain as 'worker', I think it's simpler to keep domain for scheduler(it's the domain for the top level dir we're working under).